llvm/unittests/Transforms/Instrumentation/MemProfUseTest.cpp

```diff
@@ -101,4 +101,116 @@ declare !dbg !19 void @_Z2f3v()
   EXPECT_THAT(CallSites[2],
               Pair(FieldsAre(2U, 9U), IndexedMemProfRecord::getGUID("_Z2f3v")));
 }
+
+TEST(MemProf, ExtractDirectCallsFromIRInline) {
+  // The following IR is generated from:
+  //
+  // void f1();
+  // static inline void f2() { f1(); }
+  // static inline void f3() { f2(); }
+  //
+  // void g1();
```

Comment:
Ok, we don't define the function intentionally so that there is nothing to inline.

Reply:
Yes.